sidecred.go

```diff
@@ -36,6 +44,7 @@ func (r *CredentialRequest) UnmarshalConfig(target interface{}) error {
 // hasValidCredentials returns true if there are already valid credentials
 // for the request. This is determined by the last resource state.
 func (r *CredentialRequest) hasValidCredentials(resource *Resource, rotationWindow time.Duration) bool {
+	now := time.Now()
```

Comment:
I prefer just leaving `.Before(time.Now())` since we are not reusing `now` anywhere else

Reply:
Right, sorry about that, it was a bi-product of a refactor, I had at one point had two checks using the same time comparison.